### ISSUE TITLE
Compilation of a list of interview partners

### ISSUE THREAD

**Issue by Ly0n** [protontypes/open-sustainable-technology#140]
For the next report we need a list of people we want to interviews. Criteria for selection should be:
1. High impact in the ecosystem. Long time experience.
2. Developers or users of tools and associated data sets.
3. Variety of organisation perspectives: Academia, For-Profit, Non-Profit Background, Startup, Community,...
4. Variety of topics: Biosphere, Water Suppy, Battery, ....
5. Variety of personal backgrounds: Origin, age, opinions,...

One way to find such interview partners is the list created by @andrew. You will find the contributors commited to the most projects here on the right side under committers:
https://summary.ecosyste.ms/collections/1

**Comment by andrew** [comment]
I can output that data as CSV if it's easier to consume?

**Comment by Ly0n** [comment]
@andrew Yes, that would be great. Maybe we could also combine this with this issue: #144

**Comment by jmertic** [comment]
@dbrownlf thinking you will be interested in this one

**Comment by dbrownlf** [comment]
We have some great folks involved with LF Energy who would fit the bill here so I can certainly get the project in touch with them. I'm also happy to review the broader list and provide feedback on the folks you are considering. In cases where you may want to speak to a specific person but don't have their contact info, we may also be able to help track that down.

**Comment by andrew** [comment]
@Ly0n I've put together an experimental page that summarizes all the committers to repositories in the OST project list with obvious bots hidden: https://summary.ecosyste.ms/collections/1/committers 

<img width="1191" alt="Screenshot 2023-07-31 at 11 28 02" src="https://github.com/protontypes/open-sustainable-technology/assets/1060/963ea7bf-75b1-43d4-9c66-01d937be5a64">

I'll add the ability to export as csv/json shortly.

Pro tip, you can hover over the Unique Projects count to see a list of the project urls: 
<img width="517" alt="Screenshot 2023-07-31 at 11 28 44" src="https://github.com/protontypes/open-sustainable-technology/assets/1060/a3d1b0e3-e8dd-4c97-936a-382f7bce6649">

**Comment by andrew** [comment]
CSV export is available now: https://summary.ecosyste.ms/collections/1/committers_csv (1.6MB) 

Data is updated on a daily basis, I'm also working on connecting the dots between more committers (names and emails) and repository profiles (github, gitlab, bitbucket etc).

**Comment by dbrownlf** [comment]
This is great! I agree these look like good folks to target. I did add a new tab and listed some of the top contributors to LF Energy projects in case that's helpful. I included GitHub links and email addresses.

**Comment by Ly0n** [comment]
If we have a list of new projects since last year, we could go through those as well for people of interest. It is certainly also good to hear about the fresh experiences of open source projects that are in the making. 

The top 20 contributors with the most project commits could also be interesting for readers within the report in a cleaned up plot. The question is how informative this is if someone has worked on many projects. You could use this as one factor out of many and create a top20 of developers.

**Comment by andrew** [comment]
I've discovered a way to automatically convert an email address into a GitHub username, I've set up a script to go through and discover all the github usernames for the 15,885 committers in the 1,214 projects in the summary service, it will take quite a few hours, and not all emails will successfully convert to a username but it should help save quite a lot of manual work there, the CSV url I shared earlier will automatically update with extra details over the next couple days.
